docs/index.js

```diff
@@ -20,24 +20,35 @@ app.post("/feedback/submit", async (req, res) => {
   });
 
   const feedback = req.body
+  const today = new Date().toISOString().slice(0, 19)
+
+  // Validate URL in feedback
+  if ((typeof feedback.url !== 'string') || !feedback.url.startsWith(config.getPrimaryRoute().url)) {
```

Comment:
One thing to consider is that this doesn't account for other types of injections 

```
curl -X POST  -H "Content-Type: application/json" \                                                                                        127 х  system Node  01:25:02 PM
-d "{ \"date\":\"2023-01-23T19:22:04\", \"feedback\": \"positive\", \
\"url\": \"https://docs.platform.sh/fooobar</td> <script>alert(1)</script><td> \"}" \
https://docs.platform.sh/feedback/submit
```
BUT it looks like you have something already in place to encode the data or mitigate this type of risk on the output side?

Reply:
The data is only displayed as an escaped string, so it should be safe, but it doesn't hurt to eliminated some values that would never appear in a real URL.